src/main/java/domain/Prize.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum Prize {
+    FIRST_PLACE(6, false, 2000000000),
```

Comment:
아래와 같이 함수형 인터페이스를 활용하는 방법도 있는데
가독성은 취향인 부분이라 저는 현재도 가독성이 나쁘지않다고 생각합니다 ㅎㅎ

```suggestion
    FIRST_PLACE((matchCount, isBonusMatch) -> matchCount == 6, 2000000000),
    
    private final BiPredicate<Integer, Boolean> matchRule;
```